docs/_releases/v0.5.32.md

```diff
@@ -28,9 +28,3 @@ tag: v0.5.32
 ## 📖 Documentation
 
 - [README] add: Community Forum @amino19 (#3272)
-
-## Contributors
-
-Thank you to our contributors for making this release possible:
-@Aisuko, @Chadha93, @Nikhil-Ladha, @Shreyas220, @amino19, @debjani1413, @dhruv0000, @hexxdump, @jeantiston, @killbotXD, @kumarabd, @l5io, @leecalcote, @metonymic-smokey, @navendu-pottekkat, @piyushsingariya, @samirparhi-dev, @suhaniii, @utkarsh-pro and l5io
- 
```

Comment:
This update was necessary 💯 
Most of the releases before didn't have it but then some others did now every release is NEAT 🎉

Reply:
@leecalcote Would love to know your opinion on this 👀